README.md

```diff
@@ -791,6 +791,9 @@ best effort pods are evicted before burstable and guaranteed pods.
 * All types of pods with the annotation `descheduler.alpha.kubernetes.io/evict` are eligible for eviction. This
   annotation is used to override checks which prevent eviction and users can select which pod is evicted.
   Users should know how and if the pod will be recreated.
+  The annotation only affects internal descheduler checks.
+  The anti-disruption protection provided by the [/eviction](https://kubernetes.io/docs/concepts/scheduling-eviction/api-eviction/)
```

Comment:
just checking do you mean to have `/` to denote it's a subresource?

Reply:
yes